crypto/mbedtls/Kconfig

```diff
@@ -0,0 +1,25 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig CRYPTO_MBEDTLS
+	bool "Mbed TLS Cryptography Library"
+	default n
+	---help---
+		Enable support for Mbed TLS.
+
+if CRYPTO_MBEDTLS
+
+config MBEDTLS_VERSION
+	string "MBEDTLS Version"
+	default "2.25.0"
+
+config MBEDTLS_DEFAULT_TASK_STACKSIZE
+	int "Mbed TLS app default stack size"
+	default 8192
+
+source "$APPSDIR/crypto/mbedtls/apps/benchmark/Kconfig"
+source "$APPSDIR/crypto/mbedtls/apps/selftest/Kconfig"
```

Comment:
can we remove $APPSDIR/? since other location uses the relative path.

Reply:
The Kconfigs do not use relative paths.  If you try you get an error like this:
```
/home/bashton/nuttx/wrk/apps/crypto/mbedtls/Kconfig:23: can't open file "./apps/selftest/Kconfig"
```

Reply:
Ok, let's keep APPDIR